smac/runhistory/runhistory2epm.py

```diff
@@ -153,29 +143,31 @@ def transform(self, runhistory):
         ----------
         runhistory : smac.runhistory.runhistory.RunHistory
             runhistory of all evaluated configurations x instances
-            
+
         Returns
         -------
         X: numpy.ndarray
             configuration vector x instance features
         Y: numpy.ndarray
             cost values
         '''
-        self.logger.debug("Transform Rh into X,y format")
-        
+        self.logger.debug("Transform runhistory into X,y format")
+
         assert isinstance(runhistory, RunHistory)
 
         # consider only successfully finished runs
-        s_run_dict = self.__select_runs(rh_data=copy.deepcopy(runhistory.data),
-                                        select=RunType.SUCCESS)
+        s_run_dict = {run: runhistory.data[run] for run in runhistory.data.keys()
```

Comment:
There's a deepcopy happening in the old version. Do you have any idea on whether that has an effect?